data/acs/README.md

```diff
@@ -0,0 +1,25 @@
+# ACS Data Pipeline
+
+This folder (`/data/acs`) contains Python scripts to download and load 2018 ACS data into the CEQR database. This pipeline will likely be transferred to @NYCPlanning/EDM for ownership for future years.
+
+There is an accompanying Docker image file (`Dockerfile`) and `requirements.txt` file here to support containerized execution of the Python scripts.
+
+# Quickstart
+```
+> yarn run download-acs-data
```

Comment:
Nit:  removing the `>` would allow a user to just copy the command and paste it into terminal